### ISSUE TITLE
[BUG]: AgentNeo Dashboard Launch Error: Node.js and npm Not Found

### ISSUE THREAD

**Issue by Bharanikarthick2608** [raga-ai-hub/ragaai-catalyst#35]
# Bug Report

**Describe the Bug**
I am experiencing an issue with AgentNeo where it fails to recognize that Node.js and npm are installed, despite having them correctly set up.

**To Reproduce**
Steps or code snippets to reproduce the behavior, like:
```
1. Install AgentNeo using `pip install agentneo`
2. Run the following code:

# Create an instance of FinancialAnalysisSystem
#  analysis_system = FinancialAnalysisSystem()

# Run an analysis for Apple stock with moderate risk tolerance
# analysis_system.run_analysis("AAPL", "moderate")

# Stop the tracer when analysis is complete
# analysis_system.tracer.stop()

# from agentneo import launch_dashboard
# launch_dashboard()

3. Launch the dashboard using `launch_dashboard(port=3000)`
4. Observe the error or unexpected behavior.
```

**Expected Behavior**
The dashboard should launch successfully without any errors related to Node.js and npm.

**Actual Behavior**
When executing the command python run_agentneo.py, the application successfully processes the project, starts and completes tracing, and runs the LLM, tool, and agent functions. However, it ultimately fails with an error message indicating that Node.js and npm are required but not installed, despite both being properly installed and confirmed on the system.

**Logs and Screenshots**
![Screenshot 2024-10-14 143755](https://github.com/user-attachments/assets/35d8e02f-58eb-46d7-9334-1b763d414e41)

**Environment Details**
- **Operating System**: Windows 11
- **Python Version**: 3.10.0
- **Node.js Version**: 20.18.0
- **npm Version**: 10.8.2
- **AgentNeo Version**: agentneo==1.1b0
- **Relevant Packages**: [litellm==1.49.2,aiohttp==3.10.10]

**Comment by frazakram** [comment]
facing same issue in python 3.11.10

**Comment by kiranscaria** [comment]
Resolved in #40
